calculator/parser.py

```diff
@@ -0,0 +1,132 @@
+BINARY_OPERATORS = ["+", "-", "*", "/"]
+
+
+def binary_operator(num_1, num_2, op):
```

Comment:
You don't have to do this, but a cool approach to this would be to have a dictionary that holds these:
```
BINARY_OPERATORS = { '+' : lambda lhs, rhs : lhs + rhs, '*' : lambda lhs, rhs: lhs * rhs, '/' : lambda lhs, rhs: lhs / rhs, '-' : lambda lhs, rhs: lhs - rhs }
def binary_operator(lhs, rhs, op):
    if op in BINARY_OPERATORS:
        BINARY_OPERATORS[op](lhs, rhs)
    else:
        raise ValueError("unsupported operation {}".format(op))
```